charts/kserve-resources/templates/configmap.yaml

```diff
@@ -185,6 +185,7 @@ data:
            "localGateway" : "knative-serving/knative-local-gateway",
            "localGatewayService" : "knative-local-gateway.istio-system.svc.cluster.local",
            "ingressDomain"  : "example.com",
+           "additionalIngressDomains": ["additional-example.com", "additional-example-1.com"],
```

Comment:
also need to update the kustomize directory
https://github.com/kserve/kserve/blob/master/config/configmap/inferenceservice.yaml